python/paddle/utils/download.py

```diff
@@ -121,7 +122,8 @@ def get_path_from_url(url,
                       root_dir,
                       md5sum=None,
                       check_exist=True,
-                      decompress=True):
+                      decompress=True,
+                      method='get'):
```

Comment:
add comments for `decompress` and `method`. Add candidates for `method`

Reply:
done